Core/FileSystems/ISOFileSystem.cpp

```diff
@@ -647,6 +647,9 @@ std::vector<PSPFileInfo> ISOFileSystem::GetDirListing(const std::string &path, b
 			*exists = false;
 		return myVector;
 	}
+	if (entry == &entireISO) {
+		entry = GetFromPath("/");
+	}
```

Comment:
Actually it wouldn't, because we only want it to trigger during listings, not when trying to open the root as a file.

So, I'm going to merge this as-is.